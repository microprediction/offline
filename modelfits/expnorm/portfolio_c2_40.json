{"g1": 0.07423006215839463, "g2": 8.233586919151644, "logK": 0.6120738589633258, "loc": 0.07167814637747837, "logScale": -0.2290602834568467}