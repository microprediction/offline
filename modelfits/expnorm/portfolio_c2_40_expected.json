{"g1": 0.13523227608517252, "g2": 7.402264513957871, "logK": 0.5577658874992246, "loc": -0.058047102961097406, "logScale": -1.9114458765406699}